chrome/script.browserify.js

```diff
@@ -166,15 +166,28 @@ function keyHandler(e) {
 }
 
 function switchFocus(firstSelector, nextNodeAttr) {
-  var inputId = "search-field";
-  var newActive =
-    document.activeElement.id === inputId
+  var searchField = document.getElementById('search-field');
+  var newActive = document.activeElement !== searchField
       ? document.querySelector(firstSelector)
       : document.activeElement[nextNodeAttr];
 
   if (newActive) {
     newActive.focus();
   } else {
-    document.getElementById(inputId).focus();
+    searchField.focus();
```

Comment:
This is just a tiny change I forgot to push to the previous PR it seems. I can remove it from this one if you want.

Reply:
No need, keep it here :+1: